frontend/app/src/components/ExternalTradeForm.vue

```diff
@@ -281,6 +322,34 @@ export default class ExternalTradeForm extends Vue {
     await this.fetchPrice();
   }
 
+  @Watch('amount')
+  async onAmountChange() {
+    this.onRateChange();
+    this.onQuoteAmountChange();
+  }
+
+  @Watch('rate')
+  onRateChange() {
+    if (this.amount && this.rate && this.selectedCalculationInput === 'rate') {
+      this.quoteAmount = bigNumberify(
+        parseFloat(this.amount) * parseFloat(this.rate)
```

Comment:
just fyi, you can also make BigNumbers out of the properties and perform this operation between bignumbers the library used is [bignumber.js/](https://github.com/MikeMcl/bignumber.js/)

Reply:
Okay, will change it